src/entities/trades/swapr-v3/entities/pool.ts

```diff
@@ -0,0 +1,326 @@
+import { BigintIsh, CurrencyAmount, Price, Token } from '@uniswap/sdk-core'
+import JSBI from 'jsbi'
+import invariant from 'tiny-invariant'
+import { POOL_DEPLOYER_ADDRESS } from '../constants'
+import {
+  FeeAmount,
+  LiquidityMath,
+  NoTickDataProvider,
+  SwapMath,
+  Tick,
+  TickConstructorArgs,
+  TickDataProvider,
+  TickListDataProvider,
+  TickMath,
+  computePoolAddress,
+} from '@uniswap/v3-sdk'
+
+// used in liquidity amount math
```

Comment:
Can we improve this comment? Or maybe just removed it.

Reply:
removed it